deployment/src/main/java/io/quarkiverse/fury/deployment/FurySerializerBuildItem.java

```diff
@@ -34,4 +35,25 @@ public Class<?> getClazz() {
     public Class<? extends Serializer> getSerializer() {
         return serializer;
     }
+
+    public static List<FurySerializerBuildItem> buildItems(ClassInfo classInfo) {
+        Class<?> clazz = JandexReflection.loadClass(classInfo);
+        FurySerialization annotation = clazz.getDeclaredAnnotation(FurySerialization.class);
+        Class<?>[] classes = annotation.targetClasses();
+        int classed = annotation.classId();
+        Class<? extends Serializer> serializer = annotation.serializer();
+        if (classes.length > 1) {
+            Preconditions.checkArgument(classed == -1,
```

Comment:
I think  `serializer`  need to check as well since I suppose `serializer` should only to a specfic target class?

Reply:
Not exactly, multiple classes can share same serializer. This is common, users may write a serializer for an abstract parent class, which can be used for all subclasses serialization